CHANGELOG.md

```diff
@@ -6,6 +6,67 @@ This file is generated automatically. Add individual changelog entries to the 'c
 
 This changelog documents all notable changes to VAST and is updated on every release. Changes made since the last release are in the [changelog/unreleased directory][unreleased].
 
+## [2021.06.24-rc1]
+
+### :zap: Breaking Changes
+
+- Apache Arrow is now a required dependency of VAST. The previously deprecated  build option `-DVAST_ENABLE_ARROW=OFF` no longer exists.
+  [#1683](https://github.com/tenzir/vast/pull/1683)
+
+- VAST no longer loads static plugins by default. This makes it possible for users to use static binaries with plugins enabled on all their systems, enabling only the plugins they need for the respective machine. Static plugins and bundled dynamic plugins are now treated equally.
+  [#1703](https://github.com/tenzir/vast/pull/1703)
+
+### :warning: Changes
+
+- The VAST community chat moved from Gitter to Slack. [Join us](http://slack.tenzir.com) in the `#vast` channel for interactive discussions.
+  [#1696](https://github.com/tenzir/vast/pull/1696)
+
+- The [tenzir/vast](https://hub.docker.com/r/tenzir/vast) Docker image bundles the PCAP plugin.
+  [#1705](https://github.com/tenzir/vast/pull/1705)
+
+- VAST merges lists from configuration files. E.g., running VAST with `--plugins=some-plugin` and `vast.plugins: [other-plugin]` in the configuration now results in `some-plugin` and `other-plugin` being loaded (sorted by the usual precedence).
```

Comment:
```suggestion
- VAST merges lists from configuration files. E.g., running VAST with `--plugins=some-plugin` and `vast.plugins: [other-plugin]` in the configuration now results in both `some-plugin` and `other-plugin` being loaded (sorted by the usual precedence), instead of XXX.
```

Please describe prior behavior here, which may not be obvious to the user.